app/test_main.py

```diff
@@ -1,3 +1,73 @@
+from typing import Any
+
+import pytest
+
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (-1, -2, [0, 0])
```

Comment:
The test case `(-1, -2, [0, 0])` is testing negative values. Ensure that `get_human_age` handles negative inputs correctly. If negative values are not valid, consider raising an exception instead.